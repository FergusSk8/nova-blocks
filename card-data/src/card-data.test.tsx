import React from 'react';
import { shallow } from 'enzyme';
import DataCard from './index'
describe('DataCard', () => {
    let wrapper;
    it('Should be a create a snapshot without props', () => {
      wrapper = shallow(<DataCard />);
      expect(wrapper).toMatchSnapshot();
    });
    it("Should be create a snapshot with props", () => {
      const props = {
        title:  'Title1',
        subtitle: 'Subtitle1',
        suffix: '%',
        valueStyle: {},
        title2:  'Title2',
        subtitle2: 'Subtitle2',
        suffix2: '%',
        valueStyle2: {},
        icon: {},
      };
      wrapper = shallow(<DataCard {...props} />)
      wrapper.update();
      expect(wrapper).toMatchSnapshot();
    });
});