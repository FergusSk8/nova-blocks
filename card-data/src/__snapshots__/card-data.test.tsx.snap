// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataCard Should be a create a snapshot without props 1`] = `
<Fragment>
  <Card>
    <Row
      gutter={8}
    >
      <p>
        Porcentaje de cierre
      </p>
      <Tooltip
        arrowPointAtCenter={false}
        autoAdjustOverflow={true}
        mouseEnterDelay={0.1}
        mouseLeaveDelay={0.1}
        placement="top"
        title="Ayuda"
        transitionName="zoom-big-fast"
      >
        <img
          src="test-file-stub"
        />
      </Tooltip>
    </Row>
    <Row
      align="middle"
      gutter={8}
      justify="space-between"
    >
      <Col
        xl={8}
      >
        <withConfigConsumer(Statistic)
          suffix="%"
          value={0}
          valueStyle={
            Object {
              "fontSize": "35px",
              "fontWeight": 600,
            }
          }
        />
        <span>
          1 sesión
        </span>
      </Col>
      <Col
        xl={8}
      >
        <withConfigConsumer(Statistic)
          suffix="%"
          value={0}
          valueStyle={
            Object {
              "fontSize": "35px",
            }
          }
        />
        <span>
          + 1 sesión
        </span>
      </Col>
      <img
        src="test-file-stub"
      />
    </Row>
  </Card>
</Fragment>
`;

exports[`DataCard Should be create a snapshot with props 1`] = `
<Fragment>
  <Card>
    <Row
      gutter={8}
    >
      <p>
        Title1
      </p>
      <Tooltip
        arrowPointAtCenter={false}
        autoAdjustOverflow={true}
        mouseEnterDelay={0.1}
        mouseLeaveDelay={0.1}
        placement="top"
        title="Ayuda"
        transitionName="zoom-big-fast"
      >
        <img
          src="test-file-stub"
        />
      </Tooltip>
    </Row>
    <Row
      align="middle"
      gutter={8}
      justify="space-between"
    >
      <Col
        xl={8}
      >
        <withConfigConsumer(Statistic)
          suffix="%"
          value={0}
          valueStyle={Object {}}
        />
        <span>
          Subtitle1
        </span>
      </Col>
      <Col
        xl={8}
      >
        <withConfigConsumer(Statistic)
          suffix="%"
          value={0}
          valueStyle={Object {}}
        />
        <span>
          Subtitle2
        </span>
      </Col>
      <img
        src={Object {}}
      />
    </Row>
  </Card>
</Fragment>
`;
